Assignment-02/assinment 02-1.1.c

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+//author: Siddhnta Das
+//date of creation: 05/02/20
+int main()
+{
+int i;
+printf("enter a number \n"); //input an  integer number
+scanf("%d",&i);
+float j;
+printf("enter a number \n");              //input a decimal number
+scanf("%f",&j);
+char ch;
+printf("enter the character \n");         //input a character
+scanf("%s",&ch);

```

Comment:
This is a potential security risk, as you are reading a string into a single char. You should consider using [scanf_s()](https://linux.die.net/man/3/scanf) with _%c_ instead:

```
scanf_s("%c", &c, 1);
```